### DISCUSSION TITLE
Linux - Tablet controlling game when not focused

### DISCUSSION THREAD

**Discussion by dceit** [ppy/osu#26284] [Q&A]
### Type

Game behaviour

### Bug description

**Setup**
- x11, i3 Window Manager
- Manjaro GNU/Linux
- Using built in tablet drivers from Lazer

**What is the issue**
The game is still registering button presses / movements from the tablet when the process is not in focus. In the video I switch work spaces to a ferret GIF, when the ferret is war-dancing I am still in control of the game from another workspace. You can hear this from the audio.

When switching work spaces the games audio lowers, so it is aware I am not currently in focus. Is this intended functionality?

### Screenshots or videos

https://github.com/ppy/osu/assets/72780993/3924663b-d100-4a17-84fe-63e54db99686

### Version

Lazer 2023.1229.1

### Logs

Removed logs.

**Comment by btmxh** [comment]
I make a simple fix in ppy/osu-framework#6168, not sure if that is a good approach though.
